app/controllers/balance_transactions_controller.rb

```diff
@@ -0,0 +1,58 @@
+class BalanceTransactionsController < ApplicationController
+  before_action :authenticate_user!
+  before_action :find_balance_transaction, only: %i[edit update]
+
+  def new
+  end
+
+  def create
+    @balance_transaction = Transactions::BalanceTransactionsService.new(balance_transaction_params)
```

Comment:
@marderer why do you use `Service` at the end of services names?

Reply:
@AleksSenkou my previous "mentor" taught me so)

Reply:
@marderer hah, yeah, it depends on project and conventions. You know, I had "team lead / mentor" too and taught me to do things like that :)

From clear view (I believe) that services are mostly working with data. We use `User` model without `Model` prefix, so why we should do that for services